dot/parachain/network-bridge/wire_message.go

```diff
@@ -0,0 +1,99 @@
+package networkbridge
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/gossamer/dot/network"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+type WireMessage struct {
+	inner any
+}
+
+type WireMessageValues interface {
+	ViewUpdate | ProtocolMessage
+}
+
+func setMyVaryingDataType[Value WireMessageValues](mvdt *WireMessage, value Value) {
+	mvdt.inner = value
+}
+
+func (mvdt *WireMessage) SetValue(value any) (err error) {
+	switch value := value.(type) {
+	case ViewUpdate:
+		setMyVaryingDataType(mvdt, value)
+		return
```

Comment:
Do we need this return statement?

Reply:
Yes. I think it would have been clear from other comments.